### PR TITLE
Profile view 3/4 final

### PR DESCRIPTION
**Pull request by Uli19**
<============= Administrator View Menu ===========>
The visual structure of the administrator section, the dashboard, was made, taking figma as a base with previous creation.

Description
The creation of the view with its components, defined colors, and variables so that the backend department can continue with the preparation of the application, work was done so that the vita is structured according to the base that we have as support (figma), in addition to adding shadows in the btn, custom templates to make the edges round, images as vectors and icons for easy handling of the application.

Why are these changes necessary?
They are necessary to conclude the work later, it is part of the application as an admin view, so that the admin can monitor the application.

How have these changes been tested?
Only testing has been done within the application team, this to provide feedback on the work and fix common errors within the application

<============= Visual information ===========>
Information Visual

### CODE REVIEW

File: gradle/libs.versions.toml
Comment (IsmaelMontiel710) [inline gradle/libs.versions.toml:1]: changes versions very bad

File: local.properties
Comment (IsmaelMontiel710) [inline local.properties:1]: change sdk.dir please

**Review by IsmaelMontiel710 — CHANGES_REQUESTED** [review]

File: local.properties
Comment (IsmaelMontiel710) [reply local.properties:1]: I'm sorry I don´t saw what  change sdk.dir with other direction

**Review by Gavalencia12 — CHANGES_REQUESTED** [review]
request changes

**Review by Angel591Diaz — APPROVED** [review]

**Review by Gavalencia12 — APPROVED** [review]
approved

**Review by MeganPegueros — APPROVED** [review]
Almost good :)

**Pull request merged**